_i18n/en.yml

```diff
@@ -249,6 +249,10 @@ downloads:
   desktop: Desktop
   antivirus: Antivirus
   sourcearchive: Source Code (archive)
+  localsync: Local synchronization
+  localsyncinfo: These wallets scan the blockchain on your device and don't share any key with remote servers. They provide higher privacy, but require more time synchronizing.
```

Comment:
```suggestion
  localsyncinfo: These wallets connect to @remote-nodes and scan the @blockchain on your device, without sharing any key with the nodes. They provide higher privacy, but require more time synchronizing, since missing @blocks will be scanned only when the wallet is opened.
```